README.md

```diff
@@ -122,21 +118,51 @@ To start a CI job to build binaries off of a commit push a tag starting with the
 We will cooperates with the [lotus' releases and it's testing flows](https://github.com/filecoin-project/lotus/blob/0c91b0dc1012c3e54b305a76bb25fb68390adf9d/LOTUS_RELEASE_FLOW.md?plain=1#L50) to confirm whether a tagged release is production ready:
 
 *Non-consensus breaking changes*
-- All PRs introduce non-consensus breaking changes can be merged to master as long they have maintainers' approvals.
+- All PRs introducing non-consensus breaking changes can be merged to master as long they have maintainers' approvals.
 - Roughly on a monthly basis, lotus will integrate ffi's head in `master` branch, for it's new feature release, and carry it through the testing flows.
   - `release/lotus-vX` will be created to determine the commit that lotus integrates in the corresponding release.
 - If any bug is found during the testing, the fix should land in master then get backported to `release/lotus-vX`. The updated commit should be integrated into lotus and getting tested. Repeat the steps until it can be considered as stable.
 
-#Consensus breaking changes*
+*Consensus breaking changes*
 - Consensus breaking changes should be developed in it's own branch, (branch name is suggested to be: feature branches `feat/` or bug fix branches `bug/`). 
 - Consensus breaking changes that are scoped into the next immediate network upgrade shall land in `next` branch first. The maintainers are responsible to coordinate on when to land `next` to `master` according to lotus mandatory(network upgrade) release schedules.
 - A new dev branch should be created and contributors are responsible to rebase the branch onto `master`/`next` as needed.
 
+#### Versioning
 
+The versioning in Filecoin-FFI currently follows the Lotus versioning, this is something we will refactor in the near future. As an example, if you are cutting a release for Lotus v1.28.0-rc1, the Filecoin-FFI release will be named v1.28.0-rc1 as well.
+
+#### Release Process
+
+<details>
+  <summary>Cutting a development or release candidate release:</summary>
```

Comment:
Your call, but I think we get more value out of making this a heading

```##### Cutting a development or release candidate release```

than do section collapsing.

Reason being that it can show up in the table of contents and it's possible to hyperlink to the specific section.

Reply:
Added in https://github.com/filecoin-project/filecoin-ffi/pull/461/commits/adad951bf68a1679437d9208c4bf07c795573a15